crates/runtime/Cargo.toml

```diff
@@ -17,6 +17,12 @@ ureq.workspace = true
 wasmer.workspace = true
 wasmer-types.workspace = true
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+uuid = { version = "1.10.0", features = ["serde", "v4"] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+uuid = { version = "1.10.0", features = ["serde", "v4"] }
+
```

Comment:
these directives are no longer necessary

Reply:
Could you explain? 🙂 At present these are here to ensure that we can achieve the same functionality both through WASM and not. If these were removed, what would we do instead?

Reply:
they're the same thing, just merge it into `[dependencies]`

Reply:
Ahhh, I see. Yes - I had overlooked that in this location they ended up being the same. Well-spotted!